ci/update-jetpack.js

```diff
@@ -25,6 +26,54 @@ function incrementVersion(version) {
     return result;
 }
 
+function parseVersion( version ) {
+    return version.split(/[.-]/).map(part => isNaN(Number(part)) ? part : Number(part));
+}
+
+function compareInt(a,b) {
+    if ( a === b ) {
+        return 0;
+    }
+    if ( a > b ) {
+        return 1;
+    }
+    return -1;
+}
```

Comment:
```
return +a - +b;
```
?

Reply:
Oooh right. I felt like I am doing this more complex that it needs to be :)

Reply:
fixed